apps/mobile/src/components/ChatDetailScreen/components/VexlbotMessageItem/components/TradeChecklistMeetingLocationView.tsx

```diff
@@ -53,7 +61,25 @@ export default function TradeChecklistMeetingLocationView(): JSX.Element | null
           otherSideUsername: otherSideData.userName,
           t,
         })}
-      />
+      >
+        {!!calendarEventId && (
```

Comment:
What if user firstly agree on location and later they agree on date&time. Will they have to scroll up and click this again?